dev/docker_data/license.txt

```diff
@@ -0,0 +1,4 @@
+xinhui.li@childmind.org
```

Comment:
@sgiavasis do we want to make Xinhui be the license holder or [set one up](https://surfer.nmr.mgh.harvard.edu/registration.html) with a group email address?

Reply:
Do we have a group email address?

Reply:
We're still deciding how we'll handle this- can settle after merge.